tasks/eva_2706/detect_discordant_cluster_variant.py

```diff
@@ -0,0 +1,89 @@
+import argparse
+from collections import defaultdict
+from itertools import zip_longest
+
+from ebi_eva_common_pyutils.logger import logging_config
+from ebi_eva_common_pyutils.mongodb import MongoDatabase
+from pymongo.read_concern import ReadConcern
+
+logger = logging_config.get_logger(__name__)
+logging_config.add_stdout_handler()
+
+
+def grouper(n, iterable, fillvalue=None):
+    args = [iter(iterable)] * n
+    return zip_longest(fillvalue=fillvalue, *args)
+
+
+def compare(clustered_variants, submitted_variant_position_per_rs):
+    error = 0
+    for clustered_variant in clustered_variants:
+        if not clustered_variant:
+            continue
+        pos = f'{clustered_variant.get("contig")}:{clustered_variant.get("start")}'
+        if clustered_variant['accession'] not in submitted_variant_position_per_rs:
+            logger.error(f'No submitted variant found for rs{clustered_variant["accession"]}')
+            error += 1
```

Comment:
Do we want to count these errors in this script, or just warn?  Feels like a separate issue from the position mismatch...

Reply:
I did count them as they seemed important and they would have the same impact on a merge/split as a misplaced ssid. This said I agree that they are a separate issue and we didn't spend much time on figuring out why they occur. 
I'll add another ticket for them in JIRA.